app/components/local-navbar.hbs

```diff
@@ -11,4 +11,5 @@
   <span class="local-navbar__link">
     <LinkTo @route="notifications">Notifications</LinkTo>
   </span>
+  <img src='assets/moon.png' role="button" alt='light' id="themeLogo" class="themeLogo" {{on "click" this.changeTheme}}/>
```

Comment:
Can we have more meaning text in "alt"?
Something like "Toggle the theme" or something?

Reply:
Good idea. Done.